lib/elixir/lib/inspect/algebra.ex

```diff
@@ -65,7 +71,21 @@ defmodule Inspect.Opts do
                width: pos_integer | :infinity,
                base: :decimal | :binary | :hex | :octal,
                pretty: boolean,
-               safe: boolean}
+               safe: boolean,
+               syntax_colors: boolean | [atom: IO.ANSI.ansidata]
```

Comment:
You probably want: `syntax_colors: [{atom, IO.ANSI.ansidata}]`. Your current version means it would have keys of atom `:atom`.